src/wp-includes/block-template-utils.php

```diff
@@ -1,23 +1,481 @@
+		),
+		'front-page'     => array(
+			'title'       => _x( 'Front Page', 'Template name' ),
+			'description' => __( 'Template used to render the front page of the site, whether it displays blog posts or a static page. The front page template takes precedence over the "Home" template.' ),
+		),
+		'singular'       => array(
+			'title'       => _x( 'Singular', 'Template name' ),
+			'description' => __( 'Template used for displaying single views of the content. This template is a fallback for the Single, Post, and Page templates, which take precedence when they exist.' ),
+		),
+		'single'         => array(
+			'title'       => _x( 'Single Post', 'Template name' ),
+			'description' => __( 'Template used to display a single blog post.' ),
+		),
+		'page'           => array(
+			'title'       => _x( 'Page', 'Template name' ),
+			'description' => __( 'Template used to display individual pages.' ),
+		),
+		'archive'        => array(
+			'title'       => _x( 'Archive', 'Template name' ),
+			'description' => __( 'The archive template displays multiple entries at once. It is used as a fallback for the Category, Author, and Date templates, which take precedence when they are available.' ),
+		),
+		'author'         => array(
+			'title'       => _x( 'Author', 'Template name' ),
+			'description' => __( 'Archive template used to display a list of posts from a single author.' ),
+		),
+		'category'       => array(
+			'title'       => _x( 'Category', 'Template name' ),
+			'description' => __( 'Archive template used to display a list of posts from the same category.' ),
+		),
+		'taxonomy'       => array(
+			'title'       => _x( 'Taxonomy', 'Template name' ),
+			'description' => __( 'Archive template used to display a list of posts from the same taxonomy.' ),
+		),
+		'date'           => array(
+			'title'       => _x( 'Date', 'Template name' ),
+			'description' => __( 'Archive template used to display a list of posts from a specific date.' ),
+		),
+		'tag'            => array(
+			'title'       => _x( 'Tag', 'Template name' ),
+			'description' => __( 'Archive template used to display a list of posts with a given tag.' ),
+		),
+		'attachment'     => array(
+			'title'       => __( 'Media' ),
+			'description' => __( 'Template used to display individual media items or attachments.' ),
+		),
+		'search'         => array(
+			'title'       => _x( 'Search', 'Template name' ),
+			'description' => __( 'Template used to display search results.' ),
+		),
+		'privacy-policy' => array(
+			'title'       => __( 'Privacy Policy' ),
+			'description' => '',
+		),
+		'404'            => array(
+			'title'       => _x( '404', 'Template name' ),
+			'description' => __( 'Template shown when no content is found.' ),
+		),
+	);
+
+	/**
+	 * Filters the list of template types.
+	 *
+	 * @param array $default_template_types An array of template types, formatted as [ slug => [ title, description ] ].
+	 *
+	 * @since 5.x.x
+	 */
```

Comment:
```suggestion
	/**
	 * Filters the list of template types.
	 *
	 * @since 5.9.0
	 *
	 * @param array $default_template_types An array of template types, formatted as [ slug => [ title, description ] ].
	 */
```